lib/primitives.js

```diff
@@ -1,218 +1,258 @@
-var _ = require('lodash');
+var ArrayType, BoolType, ElementType, KeyValueType, NullType, NumberType, ObjectType, StringType, convertFromDom, convertToType, _,
```

Comment:
So I wonder – is this a standard procedure – bundling generated file with the package? Shouldn't this be generated during npm install?